.changeset/three-icons-smell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-import': patch
+---
+
+gh-4276, When importing components you will now have the ability to use non unicode characters in the entity owner field
```

Comment:
Think we can drop the issue number from here to keep them looking nice in the CHANGELOGs :)